src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCashAdvanceServiceImpl.java

```diff
@@ -83,7 +83,8 @@ public boolean isPreTripCashAdvanceRequestLine(ConcurStandardAccountingExtractDe
     @Override
     public boolean isPreTripCashAdvanceIssuedByCashAdmin(ConcurStandardAccountingExtractDetailLine saeLine) {
         if (StringUtils.isNotBlank(saeLine.getCashAdvanceTransactionType())
-                && StringUtils.equalsIgnoreCase(saeLine.getCashAdvanceTransactionType(), ConcurConstants.SAE_REQUESTED_CASH_ADVANCE_APPROVED_BY_CONCUR_ADMIN)) {
+                && (StringUtils.equalsIgnoreCase(saeLine.getCashAdvanceTransactionType(), ConcurConstants.SAE_REQUESTED_CASH_ADVANCE_APPROVED_BY_CONCUR_ADMIN)
+                    || StringUtils.equalsIgnoreCase(saeLine.getCashAdvanceTransactionType(), ConcurConstants.SAE_CASH_ADVANCE_BEING_APPLIED_TO_TRIP_REIMBURSEMENT))) {
```

Comment:
Is there a reason why the applied-to-reimbursement check is being added here? Could the caller invoke `isCashAdvanceToBeAppliedToReimbursement()` instead, or does that method's additional `cashAdvanceKeyExists()` verification mess things up?